northstar/src/runtime/process/fork.rs

```diff
@@ -0,0 +1,25 @@
+use nix::unistd::{self, Pid};
+
+use crate::runtime::error::Error;
+
+/// Fork a new process.
+///
+/// # Arguments:
+///
+/// * `f` - The closure to run in the child process.
+///
+pub fn fork_process<F>(f: F) -> nix::Result<Pid>
```

Comment:
why not just `fork`? It will always be a process...

Reply:
yeah, just wanted to signal that this is some kind of wrapper

Reply:
renamed